src/archive.ts

```diff
@@ -6,10 +6,16 @@ import { promisify } from 'util'
 import archiver, { Archiver } from 'archiver'
 import endOfStream from 'end-of-stream'
 
+import { ObjectValues } from './types/utils.js'
+
 export { Archiver as ZipArchive } from 'archiver'
 
-// TODO make enum
-export type ArchiveFormat = 'none' | 'zip'
+export const ARCHIVE_FORMAT = {
```

Comment:
you could use an `const enum ARCHIVE_FORMAT` as well

Reply:
But then we cannot export the type for external use. `const enum` does not exist at runtime.